src/NuGet.Clients/NuGet.PackageManagement.UI/NuGet.PackageManagement.UI.csproj

```diff
@@ -533,10 +533,14 @@
   <Import Project="$(BuildCommonDirectory)embedinterop.targets" />
   <Import Project="$(BuildCommonDirectory)common.targets" />
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+
+  <Target Name="CleanupTemporaryTargetAssembly">
+    <Message Text="overriding target to avoid https://github.com/NuGet/Client.Engineering/issues/311"  />
```

Comment:
I feel like this change should be a separate PR linked to 311.

Reply:
yep. was just trying to test and unblock my testing at the same time.

i now have a seperate PR: https://github.com/NuGet/NuGet.Client/pull/3536
(but i still have that change as part of this change...so builds work on CI)

fixed in next iteration.